config/webpack/paths.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
You need to remove this file from the commit completely please ;). It is now gitignored.